pkgs/test/make-wrapper/lib.nix

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Please keep it near your test for this PR. You can follow it up with a proposal to make it available to other code later, which is a different goal from testing lua, and needs different reviewers. By splitting the work, your PR will be easier to merge.